beacon_node/execution_layer/src/test_utils/mock_execution_layer.rs

```diff
@@ -100,7 +107,8 @@ impl<T: EthSpec> MockExecutionLayer<T> {
             timestamp,
             prev_randao,
             Address::repeat_byte(42),
-            // FIXME: think about how to handle different forks / withdrawals here..
+            // FIXME: think about how to handle different forks here..
+            None,
```

Comment:
I thought about this but it looks like we're only using this for a single test `produce_three_valid_pos_execution_blocks`, so may not be worth the effort?

Reply:
true. and these are merge-transition tests, and we'll likely deprecate that logic at some point. removed the FIXMEs here: 383d02079fa52029ee5ea9dc5a61140aa601c434